interface/wx/propgrid/property.h

```diff
@@ -1619,7 +1707,7 @@ class wxPGProperty : public wxObject
     int Index( const wxPGProperty* p ) const;
 
     /**
-        Use this member function to add independent (i.e. regular) children to
+        Use this member function to add independent (ie. regular) children to
```

Comment:
Same thing as for `e.g.` above (numbers are 24 and 596 in favour of `i.e.` in this case).